api/app/main.py

```
@@ -0,0 +1,710 @@
+	price_cache.start()
+	store.set_price_cache(price_cache)
+	monitor = SQLiteSwapMonitor(db_path=db_path, callback=lambda s: store.upsert_swap(s), load_history=config.kdf_load_history)
+    
+	# Backfill since given timestamp or last 24 hours to make stats available on launch
+	try:
+		if config.backfill_since is not None:
+			monitor.backfill_range(int(config.backfill_since), int(time.time()))
+		else:
+			monitor.backfill_last_hours(1)
+	except Exception:
+		pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)